templates/zerver/api/webhook-walkthrough.md

```diff
@@ -373,11 +374,14 @@ available in the Zulip product, follow these steps to prepare your pull
 request:
 
 1. Run tests including linters and ensure you have addressed any issues they
-   report. See [Testing](testing.html) and [Linters](linters.html) for details.
-2. Read through [Code styles and conventions](code-style.html) and take a look
+   report. See [Testing](http://zulip.readthedocs.io/en/latest/testing.html)
+   and [Linters](http://zulip.readthedocs.io/en/latest/linters.html) for details.
+2. Read through [Code styles and conventions](
+   http://zulip.readthedocs.io/en/latest/code-style.html) and take a look
    through your code to double-check that you've followed Zulip's guidelines.
 3. Take a look at your git history to ensure your commits have been clear and
-   logical (see [Version Control](version-control.html) for tips). If not,
+   logical (see [Version Control](
+   http://zulip.readthedocs.io/en/latest/version-control.html) for tips). If not,
```

Comment:
These http:// URLs will fail lint after you rebase; should be https://.

Reply:
http:// seems to work still.

Reply:
After rebasing?  We just added the new lint rule.

Reply:
Nvm, misread.